fml/raster_thread_merger_unittests.cc

```diff
@@ -623,5 +623,229 @@ TEST(RasterThreadMerger, SetMergeUnmergeCallback) {
   thread2.join();
 }
 
+TEST(RasterThreadMerger, MultipleMergersCanMergeSameThreadPair) {
+  fml::MessageLoop* loop1 = nullptr;
```

Comment:
You do this a few times, factor out a helper function.

Reply:
How to make a helper function to deal with the multiple local variables, maybe I should define a macro? And I also think a helper function or defining macro may decreases readability of every test function instead.

Reply:
@gooooloo Helped me to refactoring this test file. The repeated code reduced many lines. thanks @gooooloo 😁